src/controllers/admin/project-member-controller.ts

```diff
@@ -3,6 +3,7 @@ import * as ProjectMemberService from "../../services/project-member-service"
 import CustomError from "../../utils/custom-error"
 import { ValidationError } from "yup"
 import { createProjectMemberSchema } from "../../utils/validation/project-member-schema"
+import { getById } from "../../services/project-service"
```

Comment:
Import this like
`import * as ProjectService from "../../services/project-service"`